### PR TITLE
better error msg

### PR DESCRIPTION
**Pull request by sahil-lakhwani**
Signed-off-by: sahil-lakhwani <sahilakhwani@gmail.com>

### CODE REVIEW

File: pkg/pmk/common.go [hunk 1]
```diff
@@ -14,7 +15,7 @@ func setupNode(hostOS string, exec cmdexec.Executor) (err error) {
 		return err
 	}
 	if err := swapOffFstab(exec, "/etc/fstab"); err != nil {
-		return err
+		return fmt.Errorf("Unable to edit file /etc/fstab")
```
Comment (anupbarve) [inline pkg/pmk/common.go:18]: This is going to mask the actual errors and make it difficult for us to debug the failures.
Could you also add a debug log with actual err.
Same comment for other instances as well.

File: pkg/platform/centos/centos.go [hunk 2]
```diff
@@ -67,7 +67,6 @@ func (c *CentOS) checkSudo() (bool, error) {
 
 	id, err := strconv.Atoi(idS)
 	if err != nil {
-		zap.S().Info(">", err)
```
Comment (anupbarve) [inline pkg/platform/centos/centos.go:70]: Was wondering what this was all about while testing some stuff :) 
Thanks for removing.

**Review by anupbarve — CHANGES_REQUESTED** [review]
Please put the console output for the manual testing.
Please also put the TC link.

File: pkg/pmk/common.go (hunk 1 above)
Comment (sahil-lakhwani) [reply pkg/pmk/common.go:18]: @anupbarve The debug log for this is printed here https://github.com/platform9/pf9ctl/blob/622631834f501d061efcf78e14b59f67c1fc4f04/pkg/cmdexec/executor.go#L57

**Review by anupbarve — APPROVED** [review]
LGTM

**Pull request merged**